allocations_familiales/prologue.catala_fr

```diff
@@ -50,8 +50,8 @@ déclaration champ d'application PrestationsFamiliales:
   résultat âge_l512_3_2 contenu durée
   résultat régime_outre_mer_l751_1 condition
   entrée date_courante contenu date
-  entrée résidence contenu Collectivité
-  smic champ d'application Smic
+  entrée résidence contenu France.Collectivité
+  smic champ d'application Smic.Smic
 
 champ d'application PrestationsFamiliales:
```

Comment:
So back when I separated `PrestationsFamiliales` from `AllocationsFamiliales` in order for `AidesLogement` to just use `PrestationsFamiliales` and not the whole `AllocationsFamiliales`, the work was not finished. Actually, I had a dependency diamond which prohibited me to include `PrestationsFamiliales` as a separate file in `AllocationsFamiliales`, because I think both rely on `Smic`. So I left a duplicated version of the  `PrestationsFamiliales` code inside `AllocationsFamiliales`, hence this line 56. Now that we have proper module, this scope can be removed from the prologue of `AllocationsFamiliales`, as well as all of its code!

Reply:
Ah, very nice! :)

I propose to merge this as is (it works and is closer to the non-modularised version), then I'll gladly work on cleaning this up and removing the duplicated code.